lightning/src/ln/channelmanager.rs

```diff
@@ -868,17 +868,30 @@ pub struct ChannelDetails {
 	pub unspendable_punishment_reserve: Option<u64>,
 	/// The `user_channel_id` passed in to create_channel, or 0 if the channel was inbound.
 	pub user_channel_id: u64,
+	/// Our total balance.  This is the amount we would get if we close the channel.
```

Comment:
Lets re-order this to include the feerate notice immediately after this sentence, then a paragraph break for the list of things it doesn't include re: HTLCs?

Reply:
Please see if the update I just pushed is what you had in mind.